tests/Validator/Constraints/ImageUploadDimensionValidatorTest.php

```diff
@@ -86,6 +86,11 @@ public function testWithNoConstraints(): void
         $this->assertNoViolation();
     }
 
+    /**
+     * @psalm-suppress InternalMethod
+     *
+     * @see https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php#L298-L301
```

Comment:
I don't understand the comment, `class ConstraintViolationAssertion` is not used here

Reply:
the `atPath` is a call to that class uses it.

Reply:
So then, again an issue could be created.

They added a lot of assertion https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Test/ConstraintValidatorTestCase.php#L265 ; if what we're wanting to do is not possible now without using an internal class ; I would say a feature request to add extra protected method could be a nice idea.